src/gpo/assets/PowerToys.admx

```diff
@@ -287,6 +289,38 @@
         <decimal value="0" />
       </disabledValue>
     </policy>
+	<policy name="DisableAutomaticUpdateDownload" class="Both" displayName="$(string.DisableAutomaticUpdateDownload)" explainText="$(string.DisableAutomaticUpdateDownloadDescription)" key="Software\Policies\PowerToys" valueName="AllowExperimentation">
```

Comment:
@stefamsjfw
Here is the bug. 


```suggestion
	<policy name="DisableAutomaticUpdateDownload" class="Both" displayName="$(string.DisableAutomaticUpdateDownload)" explainText="$(string.DisableAutomaticUpdateDownloadDescription)" key="Software\Policies\PowerToys" valueName="AutomaticUpdateDownloadDisabled">
```
 
Are you using the admx? I only created the values manually - not a good idea when testing. 😅

Reply:
yes, testing as the end user would use it :P I'll give it another try